turingschool-examples/whats-cookin-starter-kit
# Feature/site buildout

Fixes #
## Add Site Functionality

<br>

### Description

* Is this a feature or a fix?
This feature adds some dynamic additions to the webpage.

* Where should the reviewer start?
 clone the repo, open `index.html` in your web browser.

* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?
This does not add functionality, but improves usability of the site. The changes were made as the site is still a work in progress and needs to be improved upon.

* What outside features or research did you use to implement this fix?
W3Schools and CSS-Tricks helped in setting up the drop menu.

* How should this be tested?
After cloning down the repo and opening the site (directions above) reviewer should check for basic functionality of the site. Afterwards check `index.html` and `styles.css` for syntax/styling refactor possibilities/general error checking.

<br>

***

#### Please review considerations below:

- [ ] Check `index.html` for syntax/errors/improvement possibilities
- [ ] Check `styles.css` for syntax/errors/improvement possibilities
- [ ] Check webpage is running properly
- [ ] Check on pressing "Whats Cookin?" a dropdown section appears.
- [ ] Pressing "What's Cookin?" again should remove the section; an arrow next to the text should change
- [ ] Hovering over a food image should brighten the image.
